src/main/java/BMSStreamSender/BMSSender.java

```diff
@@ -0,0 +1,28 @@
+package BMSStreamSender;
+
+public class BMSSender {
+
+	public static void main(String[] args) {
+		
+		System.out.println("Please enter number of BMS readings to be generate (Min->10):");
+		try {
+		String inputGenarateRange = System.console().readLine();
+		if(ValidateUserInput.isNumeric(inputGenarateRange)&&ValidateUserInput.isGreaterthanMin(inputGenarateRange))
+		{
+			System.out.println("You entered string " + Integer.parseInt(inputGenarateRange));
+			IBMSService bms_s=new BMSServiceImpl();
+			bms_s.genearateBatteryReadings( Integer.parseInt(inputGenarateRange));
+			bms_s.printBatteryReadingsToConsole();
+		}
+		
+		}
+		catch(NullPointerException e)
```

Comment:
Avoid catch of a `NullPointerException` - because it will also hide exceptions that are due to programming errors. It will avoid crashing in this piece of code, but will give unpredictable results when the Null-pointer does happen